src/Widgets/IconWidget/IconWidgetEditingConfig.tsx

```diff
@@ -28,7 +24,21 @@ provideEditingConfig(IconWidget, {
       title: 'Size',
     },
   },
-  properties: ['icon', 'size', 'alignment', 'link'],
+  propertiesGroups: [
+    {
+      title: 'Icon',
+      component: (props: { widget: Widget }) => (
+        <ScrivitoBootstrapIconEditor defaultValue="box" {...props} />
+      ),
```

Comment:
Default not needed.

Reply:
I changed my mind - it's better to provide a defaultValue, where one is actually used.